src/pages/sighting/SightingCore.jsx

```diff
@@ -0,0 +1,279 @@
+
+export default function SightingCore({
+  data,
+  loading,
+  error,
+  statusCode,
+  pending,
+  id,
+}) {
+  const history = useHistory();
+  const intl = useIntl();
+  const queryClient = useQueryClient();
+
+  const fieldSchemas = useSightingFieldSchemas();
+
+  function refreshData() {
+    const queryKey = pending
+      ? getAGSQueryKey(id)
+      : getSightingQueryKey(id);
+    queryClient.invalidateQueries(queryKey);
+  }
+
+  const {
+    deleteSighting,
+    loading: deleteInProgress,
+    error: deleteSightingError,
+    setError: setDeleteSightingError,
+  } = useDeleteSighting();
+
+  const {
+    commitAgs,
+    isLoading: commitAgsLoading,
+  } = useCommitAssetGroupSighting();
+
+  /*
+    known issue: if data or fieldschemas change values
+    or properties this may not update
+    switch to data.version?
+  */
+  const metadata = useMemo(
+    () => {
+      if (!data || !fieldSchemas) return null;
+      return fieldSchemas.map(schema => ({
+        ...schema,
+        value: schema.getValue(schema, data),
+      }));
+    },
+    [data, fieldSchemas],
+  );
+
+  // fetch data for Id...
+  useDocumentTitle(`Sighting ${id}`, { translateMessage: false });
+
+  const [historyOpen, setHistoryOpen] = useState(false);
+  const [deleteDialogOpen, setDeleteDialogOpen] = useState(false);
+
+  const activeTab = window.location.hash || '#overview';
+
+  if (loading) return <LoadingScreen />;
+  if (statusCode === 404)
+    return (
+      <SadScreen
+        subtitleId="SIGHTING_NOT_FOUND"
+        descriptionId="SIGHTING_NOT_FOUND_DESCRIPTION"
+        variant="genericError"
+      />
+    );
+  if (error) return <SadScreen variant="genericError" />;
+  if (!data)
+    return (
+      <SadScreen
+        variant="notFoundOcean"
+        subtitleId="SIGHTING_NOT_FOUND"
+      />
+    );
+
+  const assets = get(data, 'assets', []);
+  const annotations = flatten(
+    assets.map(asset => get(asset, 'annotations')),
+  );
+  const allAnnotationsAssigned = every(annotations, 'encounter_guid');
+  const detectionComplete = get(data, 'stage') !== 'detection';
+  const showCommitAlert =
+    pending && detectionComplete && allAnnotationsAssigned;
+
+  const sightingDisplayDate = get(data, ['startTime']);
+  // const encounters = get(data, ['encounters'], []);
+
+  const sightingCreator = data?.creator;
+  const creatorName =
+    sightingCreator?.full_name ||
+    intl.formatMessage({ id: 'UNNAMED_USER' });
+  const creatorUrl = `/users/${sightingCreator?.guid}`;
+
+  return (
+    <MainColumn fullWidth>
+      <SightingHistoryDialog
+        open={historyOpen}
+        onClose={() => setHistoryOpen(false)}
+      />
+      <ConfirmDelete
+        open={deleteDialogOpen}
+        onClose={() => setDeleteDialogOpen(false)}
+        onDelete={async () => {
+          const successful = await deleteSighting(id);
+          if (successful) {
+            setDeleteDialogOpen(false);
+            history.push('/');
```

Comment:
Why this?

Reply:
Sends them back to their home page (since once the sighting is deleted the page they were on no longer exists)